frontend/src/scenes/trends/personsModalLogic.tsx

```diff
@@ -262,6 +272,20 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
 
                 return peopleResult
             },
+            loadPeopleFromUrl: async ({ url, stepNumber, breakdown, breakdown_value, label }) => {
+                const people = await (await fetch(url)).json()
```

Comment:
`await api.get(...)` would be simpler/consistent here

Reply:
Have updated